functionalDB/functionalDB.markdown

```diff
@@ -843,6 +907,28 @@ and the following structure for the `:where` part. (Each cell in the _Predicate
 </td>
 </tr>
 </table>
+: \label{500l.functionaldb.clauses} Clauses
+</markdown>
+<latex>
+\begin{table}
+\centering
+{\footnotesize
+\rowcolors{2}{TableOdd}{TableEven}
+\begin{tabular}{lll}
+\hline
+\textbf{Query Clause} & \textbf{Predicate Clause} & \textbf{Meta Clause} \\
+\hline
+\verb|[?e  :likes "pizza"]| & \verb|[#(= % %)  #(= % :likes)  #(= % "pizza")]| & \verb|["?e" nil nil]| \\
+\verb|[?e  :name  ?nm]| & \verb|[#(= % %)  #(= % :name) #(= % %)]| & \verb|["?e" nil "?nm"]| \\
+\verb|[?e  :speak "English"]| & \verb|[#(= % %) #(= % :speak) #(= % "English")]| & \verb|["?e" nil nil]| \\
+\verb|[?e  :birthday (birthday-this-month? ?bd)]| & \verb|[#(= % %) #(= % :birthday) #(birthday-this-month? %)]| & \verb|["?e" nil "?bd"]| \\

```

Comment:
This row is way too wide in the print version. Would it hurt the clarity much here to change all instances of `birthday` to `bday`, and to name the predicate `bday-this-mo`? (We'd probably have to make this change in the other tables / examples also for consistency.) If you have a better suggestion, I'm open to it :)